go-apps/meep-app-enablement/server/service-mgmt/service-mgmt.go

```diff
@@ -941,6 +1026,25 @@ func populateServiceInfoList(key string, jsonInfo string, sInfoList interface{})
 	return nil
 }
 
+func populateSubscriptionsList(key string, jsonInfo string, data interface{}) error {
+	// Get query params & userlist from user data
+	subscriptionLinkList := data.(*MecServiceMgmtApiSubscriptionLinkList)
+	if data == nil {
+		return errors.New("subscriptionLinkList not found")
+	}
+
+	// Retrieve user info from DB
+	var subscription MecServiceMgmtApiSubscriptionLinkListSubscription
```

Comment:
This code will not work, because the subscriptionType is stored in the subscription, but this value must be set in the Rel object of the subscriptionList... there was an error in the spec... but i didn't change the spec... i just went with it.. which explain the previous code...

Reply:
subscriptionType will be missing in the response...

Reply:
Good finding! I had assumed the stored subscription was returned directly. I will make the necessary fixes.